test/unittests/OpModel/TTNN/Op/TestOpModelInterface.cpp

```diff
@@ -2701,9 +2704,12 @@ TEST_F(OpModelBase, conv2dInterfaceComputeKernelConfig) {
   ASSERT_TRUE(static_cast<bool>(constraintsExp));
   const auto &[cbSize, l1PeakSize, totalPeakSize, outputSize, outputLayout] =
       constraintsExp.get();
+
+  // TODO: consider updating these values when this issue is resolved:
```

Comment:
:warning: **google\-readability\-todo** :warning:
missing username/bug in TODO
```suggestion
  // TODO(nsmith): consider updating these values when this issue is resolved:
```